cardano-cli/src/Cardano/CLI/Legacy/Run/Governance.hs

```diff
@@ -35,6 +36,8 @@ runLegacyGovernanceCmds = \case
     runLegacyGovernanceGenesisKeyDelegationCertificate sbe genVk genDelegVk vrfVk out
   GovernanceUpdateProposal out eNo genVKeys ppUp mCostModelFp ->
     runLegacyGovernanceUpdateProposal out eNo genVKeys ppUp mCostModelFp
+  GovernanceLegacyPoll sub ->
+    runGovernancePollCmds sub
```

Comment:
We should call `runLegacyGovernancePollCmds`.  The individual legacy poll commands should be the ones delegating to the era-based version.

Reply:
Legacy commands now delegate to the era-based version :+1: